core/tests/webdriverio_utils/LearnerDashboardPage.js

```diff
@@ -0,0 +1,265 @@
+      `.e2e-test-completed-goals-topic-name*=${name}`);
+    expect(await action.getText('Topic Name', topicName)).toMatch(name);
+  };
+
+  this.expectNumberOfTopicsInSuggestedForYou = async function(value) {
+    var topicNamesInLearnerTopicSummaryTiles = await $$(
+      '.e2e-test-learner-topic-summary-tile-title');
+    if (value > 0) {
+      await waitFor.visibilityOf(
+        topicNamesInLearnerTopicSummaryTiles[0],
+        'Learner Topic Name takes too long to appear');
+    }
+    expect(topicNamesInLearnerTopicSummaryTiles.length).toEqual(value);
+  };
+
+  this.expectNumberOfStoriesInCompletedStory = async function(value) {
+    var storyNamesInLearnerStorySummaryTiles = await $$(
+      '.e2e-test-story-name-in-learner-story-summary-tile');
+    if (value > 0) {
+      await waitFor.visibilityOf(
+        storyNamesInLearnerStorySummaryTiles[0],
+        'Story Name Card takes too long to appear');
+    }
+    expect(storyNamesInLearnerStorySummaryTiles.length).toEqual(value);
+  };
+
+  this.expectNumberOfTopicsInContinueWhereYouLeftOff = async function(value) {
+    var topicNamesInLearnerStorySummaryTiles = await $$(
+      '.e2e-test-topic-name-in-learner-story-summary-tile');
+    if (value > 0) {
+      await waitFor.visibilityOf(
+        topicNamesInLearnerStorySummaryTiles[0],
+        'Topic Name Card takes too long to appear');
+    }
+    expect(topicNamesInLearnerStorySummaryTiles.length).toEqual(value);
+  };
+
+  this.addTopicToLearnerGoals = async function() {
+    await action.click('Add to learner goals button', addToLearnerGoalsButton);
+  };
+
+  this.expectSubscriptionFirstNameToMatch = async function(name) {
+    await subscriptionSection.scrollIntoView();
+    var subscriptionName = await subscriptionNameSelector();
+    await waitFor.visibilityOf(
+      subscriptionName[0],
+      'Subscription First Name takes too long to appear');
+    expect(await subscriptionName[0].getText()).toMatch(name);
+  };
+
+  this.expectSubscriptionLastNameToMatch = async function(name) {
+    var subscriptionName = await subscriptionNameSelector();
+    var lastElement = subscriptionName.length - 1;
+    await waitFor.visibilityOf(
+      subscriptionName[lastElement],
+      'Subscription Last Name takes too long to appear');
+    expect(await subscriptionName[lastElement].getText()).toMatch(name);
+  };
+
+  this.expectFeedbackExplorationTitleToMatch = async function(title) {
+    var feedbackExplorationTitle = await $$(
+      '.e2e-test-feedback-exploration');
+    await waitFor.visibilityOf(
+      feedbackExplorationTitle[0],
+      'Feedback Exploration Title takes too long to appear');
+    expect(await feedbackExplorationTitle[0].getText()).toMatch(title);
+  };
+
+  this.expectFeedbackMessageToMatch = async function(message) {
+    var feedbackMessage = await $('.e2e-test-feedback-message');
+    await waitFor.visibilityOf(
+      feedbackMessage, 'Feedback Message takes too long to appear');
```

Comment:
It was earlier referenced as `feedbackMessage.first()`?

Reply:
There was only one feedback message available and if I am using multiple selectors for selecting a element which is present only at one place then its not working. I tried looking for official documentation regarding this but I didn't found one.